test/widgets/action_sheet_test.dart

```diff
@@ -143,6 +144,35 @@ void main() {
       connection = store.connection as FakeApiConnection;
     }
 
+    Future<void> showFromInbox(WidgetTester tester, {
+      String topic = someTopic,
+    }) async {
+      assert(() {
+        final channelIdsWithUnreads = store.unreads.streams.keys;
+        final topicWithUnreads = channelIdsWithUnreads.firstWhereOrNull((streamId) =>
+          store.unreads.countInTopicNarrow(streamId, TopicName(topic)) > 0);
+        if (topicWithUnreads == null) {
+          throw FlutterError.fromParts([
+            ErrorSummary('showFromInbox called without an unread message'),
+            ErrorHint(
+              'Before calling showFromInbox, ensure that [Unreads] '
+              'has an unread message in the relevant topic. ',
+            ),
```

Comment:
The error looks useful! Would it be necessary to wrap it in the `assert((){})`, though, as this only gets run in tests?

Reply:
Oh indeed; I think it doesn't need the `assert`.